tools/src/main/java/org/apache/kafka/tools/consumer/group/ConsumerGroupCommand.java

```diff
@@ -1054,7 +1054,7 @@ private List<TopicPartition> filterNonExistentPartitions(Collection<TopicPartiti
                                 .map(partitionInfo -> new TopicPartition(entry.getKey(), partitionInfo.partition())))
                         .toList();
 
-                return topicPartitions.stream().filter(element -> !existPartitions.contains(element)).toList();
+                return topicPartitions.stream().filter(tp -> !existPartitions.contains(tp)).toList();
```

Comment:
nit: I think this rename is a bit unhelpful because "tp" does at least have a gentle implication of the type of the elements.